src/app/tabs/test-tab.js

```diff
@@ -141,37 +162,117 @@ module.exports = class TestTab extends ViewPlugin {
     // result.passingNum
     // result.failureNum
     // result.timePassed
-    this.testsSummary.hidden = false
     cb()
   }
 
+  cleanFileName (fileName, testSuite) {
+    return fileName ? fileName.replace(/\//g, '_').replace(/\./g, '_') + testSuite : fileName
+  }
+
+  setHeader (status) {
+    if (status) {
+      const label = yo`
+        <div
+        class="alert-success d-inline-block mb-1 mr-1 p-1 passed_${this.runningTestFileName}"
```

Comment:
could you please add a tab for everything between <div></div> ?

Reply:
Yeah, sure!